java/system/ksp-allow-java-jboss-configs.yaml

```diff
@@ -0,0 +1,141 @@
+      - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/bin/java
+    - path: /opt/jboss/keycloak/modules/system/layers/base/com/fasterxml/jackson/core/jackson-core/main/module.xml
+      fromSource:
+      - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/bin/java
+    - path: /opt/jboss/keycloak/modules/system/layers/base/javax/api/main/module.xml
+      fromSource:
+      - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/bin/java
+    - path: /opt/jboss/keycloak/modules/system/layers/base/org/aesh/main/module.xml
+      fromSource:
+      - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/bin/java
+    - path: /opt/jboss/keycloak/modules/system/layers/base/org/aesh/main/readline-2.0.jar
+      fromSource:
+      - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/bin/java
+    - path: /opt/jboss/keycloak/modules/system/layers/base/org/jboss/as/domain-management/main/module.xml
+      fromSource:
+      - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/bin/java
+    - path: /opt/jboss/keycloak/modules/system/layers/base/org/jboss/logging/main/jboss-logging-3.4.1.Final.jar
+      fromSource:
+      - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/bin/java
+    - path: /opt/jboss/keycloak/modules/system/layers/keycloak/com/github/ua-parser/main/module.xml
+      fromSource:
+      - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/bin/java
+    - path: /opt/jboss/keycloak/standalone/configuration/standalone_xml_history
+      fromSource:
+      - path: /usr/bin/rm
+    - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/conf/logging.properties
+      fromSource:
+      - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/bin/java
+    - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/conf/security/java.policy
+      fromSource:
+      - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/bin/java
+    - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/conf/security/java.security
+      fromSource:
+      - path: /usr/lib/jvm/java-11-openjdk-11.0.7.10-1.el8_1.x86_64/bin/java
+    - path: /usr/lib/locale/en_US.utf8/LC_CTYPE
+      fromSource:
+      - path: /usr/bin/rm
+    - path: /usr/lib/locale/en_US.utf8/LC_MESSAGES/SYS_LC_MESSAGES
+      fromSource:
+      - path: /usr/bin/rm
+      - path: /usr/bin/sed
+    - path: /usr/lib/locale/en_US.utf8/LC_MONETARY
+      fromSource:
+      - path: /usr/bin/rm
+      - path: /usr/bin/sed
+    - path: /usr/lib/locale/en_US.utf8/LC_NUMERIC
+      fromSource:
+      - path: /usr/bin/sed
+    - path: /usr/lib64/gconv/gconv-modules.cache
+      fromSource:
+      - path: /usr/bin/coreutils
+      - path: /usr/bin/grep
+    - path: /usr/share/locale/locale.alias
+      fromSource:
+      - path: /usr/bin/tr
+    matchDirectories:
+    - dir: /lib64/
+      fromSource:
+      - path: /usr/bin/coreutils
+      - path: /usr/bin/openssl
+      - path: /usr/bin/rm
```

Comment:
@tamilmaran-7 Please re-check that you are getting these policies within the correct application workflow.